angular/src/components/settings/vault-timeout-input.component.ts

```diff
@@ -132,6 +133,10 @@ export class VaultTimeoutInputComponent implements ControlValueAccessor, Validat
       return { policyError: true };
     }
 
+    if (control.value < VaultTimeoutInputComponent.MIN_CUSTOM_MINUTES) {
```

Comment:
You also want to check that the user has selected a custom value. e.g. The "on refresh" option is represented by `-1`: https://github.com/bitwarden/web/blob/master/src/app/settings/preferences.component.ts#L46. Desktop also has some nonstandard options that are negative values. They are special cases and we still want to allow them.

A custom value is indicated by `this.form.vaultTimeout` having the `CUSTOM_VALUE` value.

Reply:
Actually, this `validate` method applies at the form level and is only exposed to the parent component. You need access to this validation state within this component so that you can show the inline error message. I suggest you extract this logic to a new method, that you can call within the component (to decide whether to show the in-line validation error) and also from this method (to decide whether the form as a whole is valid).